src/structs/Object.jl

```diff
@@ -110,6 +114,41 @@ function Object(frames, func::Function, start_pos::Union{Object,Point}; kwargs..
     return object
 end
 
+"""
+    getjpaths!(obj::Object, func::Function , args=[])
+
+This is called at rendertime . `getjpaths!` updates `obj.jpaths` with jpaths that
+are generated by drawing `func` . Typically `func` is `obj.opts[:original_func]`
+"""
+function getjpaths!(obj::Object, func::Function, args = [])
+    m = getmatrix()
+    setmatrix([1.0, 0, 0, 1.0, 0, 0])
+    empty!(CURRENT_JPATHS)
+    global CURRENT_FETCHPATH_STATE = true
+    global DISABLE_LUXOR_DRAW = true
+    v, o, f = nothing, nothing, nothing
```

Comment:
Hi, can't you pass `v, o, f` explicitly as arguments to `getjpaths` instead of defining them as `nothing, nothing, nothing`? It looks they are always available where `getjpaths!` is called